db/memtable_list.cc

```diff
@@ -383,6 +383,14 @@ void MemTableList::PickMemtablesToFlush(uint64_t max_memtable_id,
             });
 }
 
+void MemTableList::ExportMemtables(autovector<MemTable*>* ret) {
+  const auto& memlist = current_->memlist_;
+  for (auto it = memlist.rbegin(); it != memlist.rend(); ++it) {
```

Comment:
I can see the list is sorted in `PickMemtablesToFlush`. Should here also be sorted?